Source/ACE.Server/WorldObjects/Player_Bank.cs

```diff
@@ -135,6 +135,76 @@ public void DepositLegendaryKeys()
                         break;
                     }
                 }
+
+                var legeventKeysList = this.GetInventoryItemsOfWCID(500010); //25-use Legendary keys
+                foreach (var legevent in legeventKeysList)
+                {
+                    if (this.TryConsumeFromInventoryWithNetworking(legevent))
+                    {
+                        BankedLegendaryKeys += legevent.Structure ?? 25;
+                    }
+                    else
+                    {
+                        break;
+                    }
+                }
+            }
+        }
+
+        public void DepositPeas()
+        {
+            if (BankedPeas == null)
```

Comment:
I think this should also be BankedPyreals. The null check just exists because if it's the first time they deposit to that property it's null, not zero, so this sets it for safety